src/ios/RealmJS.xcodeproj/project.pbxproj

```diff
@@ -246,7 +246,6 @@
 			children = (
 				F6BCCFDF1C83809A00FE31AE /* lib */,
 				F62A35131C18E6E2004A917D /* iOS */,
-				F60103051CC4ADE500EC01BA /* JS */,
```

Comment:
Why did you remove this group? It was helpful for me to keep the JS engine abstractions in a separate group from the stuff that uses those abstractions.